src/Storage/Controllers/InstancesController.cs

```diff
@@ -136,6 +136,11 @@ public class InstancesController : ControllerBase
             string orgClaim = User.GetOrg();
             int? userId = User.GetUserIdAsInt();
 
+            if (instanceOwnerPartyId.HasValue && !string.IsNullOrEmpty(instanceOwnerIdentifier))
+            {
+                return BadRequest("Both InstanceOwnerPartyId and InstanceOwnerIdentifier cannot be present at the same time.");
```

Comment:
Adding the '.' to get the same notation as query parameter
```suggestion
                return BadRequest("Both InstanceOwner.PartyId and InstanceOwnerIdentifier cannot be present at the same time.");
```